lib/Cake/TestSuite/ControllerTestCase.php

```diff
@@ -418,4 +418,18 @@ public function generate($controller, $mocks = array()) {
 		return $this->controller;
 	}
 
+/**
+ * Unsets some properties to free memory.
+ *
+ * @return void
+ */
+	public function tearDown() {
+		parent::tearDown();
+		unset($this->contents);
+		unset($this->controller);
+		unset($this->headers);
+		unset($this->result);
+		unset($this->view);
+		unset($this->vars);
```

Comment:
You could combine all of this into one unset() as well.

Reply:
Of course if that is a preferable syntax. One line or multiple?